dotCMS/src/main/java/com/dotcms/publisher/pusher/wrapper/StructureWrapper.java

```diff
@@ -63,12 +63,12 @@ public void setOperation(Operation operation) {
         this.operation = operation;
     }
 
-    public String getWorkflowSchemaId() {
-        return workflowSchemaId;
+    public List<String> getWorkflowSchemaIds() {
+        return workflowSchemaIds;
     }
 
-    public void setWorkflowSchemaId(String workflowSchemaId) {
-        this.workflowSchemaId = workflowSchemaId;
+    public void setWorkflowSchemaIds(List<String> workflowSchemaIds) {
+        this.workflowSchemaIds = workflowSchemaIds;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Store a copy of "workflowSchemaIds". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)